src/components/tests/CaptureFilter.test.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act, render, screen, within, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { AppProvider } from '../../context/AppContext';
+import CaptureFilter from '../CaptureFilter';
+import Verify from '../Verify';
+import { ORGS } from './fixtures';
+
+import * as loglevel from 'loglevel';
+const log = loglevel.getLogger('../tests/capturefilter.test');
+
+describe('CaptureFilter organizations', () => {
+  let api;
+
+  beforeEach(() => {
+    //mock the api
+    api = require('../../api/treeTrackerApi').default;
+
+    api.getOrganizations = () => {
+      // log.debug('mock getOrganizations:');
+      return Promise.resolve(ORGS);
+    };
+  });
+
+  describe('CaptureFilter', () => {
+    describe('w/o data in context', () => {
+      let component;
+
+      beforeEach(async () => {
+        component = (
+          <AppProvider>
+            <CaptureFilter />
+          </AppProvider>
+        );
+      });
+
+      afterEach(cleanup);
+
+      it('renders without crashing', () => {
+        const div = document.createElement('div');
+        ReactDOM.render(component, div);
+        ReactDOM.unmountComponentAtNode(div);
+      });
+
+      it('renders text "Verification Status" ', () => {
+        render(component);
+        expect(screen.getByText('Verification Status')).toBeInTheDocument();
+      });
+
+      it('renders "Start Date" input ', () => {
+        render(component);
+        let input = screen.getByRole('textbox', { name: 'Start Date' });
```

Comment:
I know it's _only_ a test script, but there are a lot of `let` statements in this file that could be `const`.